lib/modules/filteReddit.js

```diff
@@ -54,8 +54,8 @@ module.options = {
 	},
 	showFilterVisitedLinks: {
 		type: 'boolean',
-		value: process.env.BUILD_TARGET !== 'safari',
-		noconfig: process.env.BUILD_TARGET === 'safari',
+		value: process.env.BUILD_TARGET !== 'safari' || process.env.BUILD_TARGET !== 'edge',
```

Comment:
Only on the second one, this should still be `&&`, sorry

Since you want `!value == noconfig`, and `!(!a && !b) == a || b`